python/pyarrow/tests/test_extension_type.py

```diff
@@ -1317,6 +1317,54 @@ def test_tensor_type():
     assert tensor_type.dim_names == ['C', 'H', 'W']
     assert tensor_type.permutation is None
 
+    tensor_type = pa.variable_shape_tensor(pa.int8(), 2)
```

Comment:
Can you make this its own test (and rename the one above to be specific about fixed shape tensor)

Reply:
Done.